data/net/minecraft/data/recipes/RecipeProvider.mapping

```diff
@@ -1,40 +1,99 @@
 CLASS net/minecraft/data/recipes/RecipeProvider
+	METHOD <init> (Lnet/minecraft/data/DataGenerator;)V
+		ARG 1 generator
 	METHOD banner (Ljava/util/function/Consumer;Lnet/minecraft/world/level/ItemLike;Lnet/minecraft/world/level/ItemLike;)V
-		ARG 0 recipeConsumer
+		ARG 0 consumer
```

Comment:
`finishedRecipeConsumer` for all recipe consumers, I won't mention the rest in this class.